source/_integrations/solax.markdown

```diff
@@ -39,6 +39,22 @@ template:
 
 {% endraw %}
 
+### Configuring the Energy Dashboard
+
+There are generally 3 sensors from your inverter that you need to configure in the energy dashboard:
+
+- the total consumption sensor (in kWh) for the "Grid Consumption"
+- the total feed-in sensor (in kWh) for the "Return to grid", i.e. the solar panel energy that you do not consume and return to the grid instead
+- the total on-grid yield sensor (in kWh) for the "Solar production"
+
+These are "running total" sensors, do not use the "daily" sensors.
+
+Depending on the type of inverter, the sensors you need may be named differently. Here is an example with the `X1HybridGen4` inverter:
+
+- `sensor.solax_XXXXXXXX_total_consumption` for the "Grid Consumption"
+- `sensor.solax_XXXXXXXX_total_feed_in_energy` for the "Return to grid"
+- `sensor.solax_XXXXXXXX_on_grid_total_yield` for the "Solar production"
```

Comment:
We should not reference them by entity IDs.

They are not guaranteed and above all translated.

Let's remove that.

Reply:
ok the problem I am trying to solve is that  it is not obvious which entity tracks what - the naming conventions are not clear for non-electricians ;)
In my solax sensors, I have 4 different sensors that are compatible with the "Solar Production" section in the energy dashboard: which one should I pick?
Using a practical example might help make this easier for other users. WDYT?

Reply:
What we normally do is that we do not literally name the entity_id, but describe what kind of entity you have.